app/src/main/java/org/fossasia/susi/ai/device/deviceconnect/DeviceConnectPresenter.kt

```diff
@@ -61,10 +61,10 @@ class DeviceConnectPresenter(deviceActivity: DeviceActivity, manager: WifiManage
         }
         if (!list.isEmpty()) {
             deviceConnectView?.setupWiFiAdapter(connections)
-            deviceConnectView?.unregister()
+//            deviceConnectView?.unregister()
         } else {
             deviceConnectView?.onDeviceConnectionError(utilModel.getString(R.string.no_device_found), utilModel.getString(R.string.setup_tut))
-            deviceConnectView?.unregister()
+//            deviceConnectView?.unregister()
```

Comment:
If not needed, remove the comment

Reply:
It may be needed afterwards @codedsun

Reply:
Then format the comments correctly acc. to Java style guidelines